README.md

```diff
@@ -344,146 +344,124 @@ to get started setting up your self-hosted runner.
+
+For example, the following workflow
 deploys a `t2.micro` instance on AWS EC2 and trains a model on the instance.
-After the instance is idle for 120 seconds, it automatically shuts down.
+After the job runs, the instance automatically shuts down.
 
 ```yaml
-name: train-my-model
+name: "Train-in-the-cloud"
 on: [push]
+
 jobs:
-  deploy-cloud-runner:
+  deploy-runner:
     runs-on: [ubuntu-latest]
-    container: docker://dvcorg/cml:latest
     steps:
+      - uses: hashicorp/setup-terraform@v1
+      - uses: iterative/setup-cml@v1
+      - uses: actions/checkout@v2
       - name: deploy
+        shell: bash
         env:
-          repo_token: ${{ secrets.REPO_TOKEN }}
+          repo_token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         run: |
-          echo "Deploying..."
-
-          MACHINE="cml$(date +%s)"
-          docker-machine create \
-              --driver amazonec2 \
-              --amazonec2-instance-type t2.micro \
-              --amazonec2-region us-east-1 \
-              --amazonec2-zone f \
-              --amazonec2-vpc-id vpc-06bc773d85a0a04f7 \
-              --amazonec2-ssh-user ubuntu \
-              $MACHINE
-
-          eval "$(docker-machine env --shell sh $MACHINE)"
-
-          (
-          docker-machine ssh $MACHINE "sudo mkdir -p /docker_machine && sudo chmod 777 /docker_machine" && \
-          docker-machine scp -r -q ~/.docker/machine/ $MACHINE:/docker_machine && \
-
-          docker run --name runner -d \
-            -v /docker_machine/machine:/root/.docker/machine \
-            -e RUNNER_IDLE_TIMEOUT=120 \
-            -e DOCKER_MACHINE=${MACHINE} \
-            -e RUNNER_LABELS=cml \
-            -e repo_token=$repo_token \
-            -e RUNNER_REPO="https://github.com/${GITHUB_REPOSITORY}" \
-           dvcorg/cml-py3 && \
-
-          sleep 20 && echo "Deployed $MACHINE"
-          ) || (echo y | docker-machine rm $MACHINE && exit 1)
-  train:
-    needs: deploy-cloud-runner
-    runs-on: [self-hosted, cml]
-
+          cml-runner \
+          --cloud aws \
+          --cloud-region us-west \
+          --cloud-type=t2.micro \
+          --labels=cml-runner
+  train-model:
+    needs: deploy-runner
+    runs-on: [self-hosted,cml-runner]
+    container: docker://dvcorg/cml-py3
     steps:
-      - uses: actions/checkout@v2
-      - name: cml_run
-        env:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          pip install -r requirements.txt
-          python train.py
-
-          cat metrics.txt >> report.md
-          cml-publish confusion_matrix.png --md >> report.md
-          cml-send-comment report.md
+    - uses: actions/checkout@v2
+    - name: "Train my model"
+      env:
+        repo_token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
Any reason to use `GITHUB_TOKEN` above but `PERSONAL_ACCESS_TOKEN` here?

Reply:
Nope, just drowning in tokens and forgetting what i called them :)